src/modeler/AutoRest.Swagger/Model/ServiceDefinition.cs

```diff
@@ -79,6 +79,8 @@ public ServiceDefinition()
         /// Key is actual path and the value is serializationProperty of http operations and operation objects.
         /// </summary>
         [Rule(typeof(UniqueResourcePaths))]
+        [Rule(typeof(ListOperationNamingWarning))]
+        [Rule(typeof(ListByOperationsValidation))]
```

Comment:
general question: shouldn't ALL of these also apply to `CustomPaths`??? Check with the decision makers